lms/views/lti/basic_launch.py

```diff
@@ -61,11 +61,15 @@ def __init__(self, context, request):
     def configured_launch(self):
         """Display a document if we can resolve one to show."""
 
-        return self._show_document(
+        self._show_document(
             document_url=self.request.find_service(DocumentURLService).get_document_url(
                 self.request
             )
         )
+        self.request.registry.notify(
+            LTIEvent(self.request, EventType.Type.CONFIGURED_LAUNCH)
```

Comment:
It might be nice to duplicate the `Type` on the `LTIEvent` object so you can do:

```LTIEvent(self.request, LTIEvent.Type...)```

And not have to worry about a random import.